man/docker-cp.1.md

```diff
@@ -8,6 +8,9 @@ docker-cp - Copy files/folders between a container and the local filesystem.
 **docker cp**
 [**--help**]
 CONTAINER:PATH LOCALPATH|-
+
+**docker cp**
+[**--help**]
```

Comment:
Not quite the same, final look after change: 
```
       docker cp [--help] CONTAINER:PATH LOCALPATH|-

       docker cp [--help] LOCALPATH|- CONTAINER:PATH
```
And original is as following:
```
 docker cp [--help] CONTAINER:PATH LOCALPATH|- LOCALPATH|- CONTAINER:PATH
```

It's hard to understand for me, I think the change makes it clearer, WDYT? :smile: